client/ayon_blender/api/plugin.py

```diff
@@ -396,6 +391,44 @@ def set_instance_data(
             }
         )
 
+    def _transfer_workfile_property(
+            self, node: bpy.types.Collection)-> bpy.types.Collection:
+        """Transfer all workfile-related instance data from
+        AYON_CONTAINERS to AYON_INSTANCES if any. Backward
+        compatibility only.
+
+        Args:
+            node (bpy.types.Collection): instance node
+
+        Returns:
+            bpy.types.Collection: instance node
+        """
+        if (
+            not isinstance(node, bpy.types.Collection)
+            or node.name != AYON_CONTAINERS
+            or AYON_PROPERTY not in node
+        ):
+            return node
+
+        instance_node = bpy.data.collections.get(AYON_INSTANCES)
+        if not instance_node:
+            instance_node = self._create_instance_node()
+        instance_node[AYON_PROPERTY] = node.get(AYON_PROPERTY)
+        del node[AYON_PROPERTY]
+        return instance_node
```

Comment:
This is Workfile Creator related - it should not be on the base `BlenderCreator` plug-in.